discussion/app/discussion/model/DiscussionKey.scala

```diff
@@ -18,7 +18,7 @@ object DiscussionKey {
     new PathBindable[DiscussionKey] {
       override def bind(key: String, value: String): Either[String, DiscussionKey] = {
         for {
-          keyAsString <- stringBinder.bind(key, value).right
+          keyAsString <- stringBinder.bind(key, value)
```

Comment:
From the docs: "Either is right-biased, which means that Right is assumed to be the default case to operate on." (https://www.scala-lang.org/api/current/scala/util/Either.html)